README.md

```diff
@@ -29,6 +29,8 @@ releases:
   # Published chart example
   - name: vault                            # name of this release
     namespace: vault                       # target namespace
+    tags:                                  # Arbitrary key value pairs for filtering releases
```

Comment:
What do you think about using `labels` instead of `tags`? would be consistent with k8s.

Reply:
Sounds good in general.

One thing I'm wondering on top of that though is - Would it also be reasonable to rename the `--tags` flag to `--selector` for more consistency with k8s'?

And in that case, I can't help expecting that `labels` could be somehow propagated as `values` to helm charts, so that we can use the same set of `labels` to filter both releases(`helmfile --selector ...`) and resulting k8s objects(`kubectl get --selector ...`). What do you think about that?

Reply:
I like the `-l`,`--selector` standardization. I'll switch to that. As far as propagating labels to the resulting k8s objects. I like the idea. I'm not sure how the implementation would work exactly since the objects would be managed via helm but is out of scope for this PR